skin/light/light.css

```diff
@@ -178,6 +175,44 @@
   color: #333 !important;
 }
 
+.tabbrowser-tab.only-fade-in {
+  min-height: 36px;
+  max-height: 36px;
+  animation: 0.5s fade-in;
+  animation-fill-mode: forwards;
+}
+
+.tabbrowser-tab.tab-hidden {
+  min-height:36px;
+  animation: 0.5s fade-out, 0.5s 0.5s slide-out;
```

Comment:
So, to make sure we're delaying the right amount, I think we should use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables) for the first animation time (which will also be the delay), and the second animation time.